libraries/provider_hab_service_systemd.rb

```diff
@@ -24,9 +24,6 @@ class HabServiceSystemd < Chef::Provider::Service
         use_inline_resources
 
         provides :hab_service_systemd
-        provides :hab_service do |node|
-          Chef::Platform::ServiceHelpers.service_resource_providers.include?(:systemd)
-        end
```

Comment:
why are you removing this?

and you should remove the matcher as well if its gone.